source/GettingStarted.rst

```diff
@@ -28,7 +28,7 @@ Users with the Administrator role can perform the steps below.
 Users with the Principal Investigator role can perform the steps below.
   
   * `Adding a New Project`_
-  *  Assigning Researchers to projects
+  * `Assigning Researchers to projects`_
```

Comment:
Does this hyper-link work? I removed the hyper-link because it was not leading anywhere.

Reply:
I added data for that. So I make it a link.